server/api.go

```diff
@@ -100,9 +103,13 @@ func (d dexAPI) CreatePassword(ctx context.Context, req *api.CreatePasswordReq)
 		return nil, errors.New("no user ID supplied")
 	}
 	if req.Password.Hash != nil {
-		if err := checkCost(req.Password.Hash); err != nil {
+		cost, err := checkCost(req.Password.Hash)
+		if err != nil {
 			return nil, err
 		}
+		if cost > recCost {
+			d.logger.Warnln("password encryption might take awhile; bcrypt cost was", cost)
```

Comment:
Could you change this message to something like "Bcrypt cost = %d, password encryption might timeout. Recommended bcrypt cost is 12."